auto_submit/lib/model/pr_type.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+enum PullRequestType {
```

Comment:
I'm a bit confused by the approaches here. It seems like this details the merge methods, but then it also lists revert.

Can you add docs calrfiying?

Reply:
Yeah this is a bit confusing. I think we really only care about the merge and revert as the two types.